quickevent/app/quickevent/plugins/Event/src/services/emmaclientwidget.cpp

```diff
@@ -88,9 +95,13 @@ bool EmmaClientWidget::saveSettings()
 		ss.setExportDir(dir);
 		ss.setExportIntervalSec(ui->edExportInterval->value());
 		ss.setFileName(ui->edFileName->text().trimmed());
-		ss.setExportStart(ui->chExportStart->isChecked());
-		ss.setExportFinish(ui->chExportFinish->isChecked());
-		ss.setExportTypeXML3(ui->chExportXML30->isChecked());;
+		ss.setExportStartTypeTxt(ui->chExportStartTxt->isChecked());
+		ss.setExportFinishTypeTxt(ui->chExportFinishTxt->isChecked());
+		ss.setExportStartTypeXML3(ui->chExportStartXML30->isChecked());;
+		ss.setExportResultTypeXML3(ui->chExportResultsXML30->isChecked());;
+		if (ss.fileName().isEmpty())
```

Comment:
nevim, jestli be se to taky nemelo jmenovat `eventName` nebo `fileNameBase`, skutecne `file name` se z toho vytvari pridanim pripony, takze je to matouci.

Reply:
fileNameBase